tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/ActionTests.cs

```diff
@@ -463,6 +527,12 @@ public async Task Action_HttpRequest()
                     .AssertReply("string")
                     .AssertReply("object")
                     .AssertReply("array")
+                    .AssertReply("VGVzdEltYWdl")
```

Comment:
The bytes array would be auto-base64-encoded by the sdk itself. So the result would be "VGVzdEltYWdl"